docs/content-update-details.md

```diff
@@ -30,7 +30,12 @@ The current list of namespaces excluded from the controller's watches:
 - openshift-sdn
 - openshift-service-ca-operator
 
-The list is not yet configurable, but most likely will become so as the project's lifecycle progresses.
+The list is configurable by informing `--ignorenamespace` to the `hostpath` plugin instance. The
+plugin can also be configured with `--refreshresources` flag, which makes the controller keep a warm
+cache of `ConfigMap` and `Secret` objects, and as they change the controller will follow the updates.
+When `--refreshresources` is disabled (i.e. `--refreshresources=false`), the controller will read
+the backing-resource (`.spec.backingResource`) just before mounting the volume instead of keeping a
+warm cache.
```

Comment:
This looks good with the current implementation, but it reminded me of something.  Something not explicitly called out in the Jira as I  revisited it, but something I always had in mind when we first created this story.

Namely, if refreshresources is disabled, we force the volume to be read only.
1) the code check would be around https://github.com/openshift/csi-driver-shared-resource/blob/7c62db45752262172549a6f4ee5fee3c605606f2/pkg/hostpath/nodeserver.go#L163-L165
2) I don't see why for the scenarios we are considering we would allow the consuming pod to change the contents we mounted, or add new files to our mount
3) It defers the need of the atomic writer stuff from k8s/k8s as we would not have to ever coordinate with the pod 
4) remember, we claim to not be a "generic" csi volume provider, so ignoring user options like read only is a legitimate stance

@adambkaplan WDYT ?

Reply:
I think it's important to highlight that when `--refreshresources` is disabled, the data on the object-cache is refreshed just before the volume is mounted. Therefore, every time the volume is mounted the cache-key would get in sync with the apiserver again.

Reply:
we have both "short living" pods like builds and will have "long running" pods which will be comsuming config maps with certs

in those long running cases, the volumes would only be mounted once

but while a good distinction to note @otaviof I think it is still orthogonal to the classic atomic write k8s secret stuff 

there are 2 main motivators IMO for this feature's original creation
1) scaling benefits from not employing a watch
2) defaulting to read only to avoid atomic write hoopla when we know we won't update content after pod creation

Reply:
@gabemontero please consider the latest changes. Now on every volume mount when refresh-resources is not enabled, it will be a read-only volume.